index.src.html

```diff
@@ -1173,7 +1154,18 @@ <h4 algorithm id="collectfromcredentialstore-passwordcredential">
   <ol class="algorithm">
     1.  Assert: |options|["{{CredentialRequestOptions/password}}"] [=map/exists=].
 
-    2.  If |options|["{{CredentialRequestOptions/password}}"] is not `true`, return the empty set.
+    2.  Return the empty set if any of the following are true:
+
+        1.  |options|["{{CredentialRequestOptions/password}}"] is not `true`.
+
+        2.  The [=current settings object=] does not have a
+            [=environment settings object/responsible document=].
+
+        3.  The [=current settings object=]'s [=environment settings object/responsible document=]
+            is not the [=active document=] in a [=top-level browsing context=].
```

Comment:
I stumbled about the "**a** [=top-level browsing context=]". Is that correct? Should this be "the" with a qualified top-level browsing context?

Reply:
The browser can have multiple top-level browsing contexts, so "the" seems overly restrictive. *shrug* Looks like the HTML spec uses "a top-level browsing context" 42 times, and "the top-level browsing context" 27 times. I think it's fine either way. :)

Reply:
About the multiple top-level browsing contexts.

What would happen if 

* www.example.com is opened in a tab
* www.foobar.com is opened in another tab with an iframe for www.example.com, in that iframe, www.example.com tries to retrieve a password credential.

Would the top-level browsing context of the first tab enable the second access?

Reply:
This seems like it would work as expected: the framed `example.com` would look at its own settings object's responsible document, and note that it is not the active document in a top-level browsing context (because it's in a frame).

The fact that `example.com` is open in another window doesn't change the state of the context actually calling the method.

Reply:
I had interpreted "is not the [=active document=] in a [=top-level browsing context=]" as "is not the [=active document=] in **any** [=top-level browsing context=]", where the latter www.example.com would be just another top-level browsing context.